src/main/webapp/src/app/recommendation-form/recommendation-form.component.spec.ts

```diff
@@ -0,0 +1,100 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { ReactiveFormsModule } from '@angular/forms';
+import { By } from '@angular/platform-browser';
+
+import { RecommendationFormComponent } from './recommendation-form.component';
+
+describe('RecommendationFormComponent', () => {
+  let component: RecommendationFormComponent;
+  let fixture: ComponentFixture<RecommendationFormComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [ RecommendationFormComponent ],
+      imports: [
+        ReactiveFormsModule
+      ]
+    })
+    .compileComponents();
+  });
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(RecommendationFormComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create the recommendation form component', () => {
+    expect(component).toBeTruthy();
+  });
+
+  it('should render heading', () => {
+    const fixture = TestBed.createComponent(RecommendationFormComponent);
```

Comment:
For all your tests, you don't need to create the fixture because it has already been created in the beforeEach (unless you need to create a different one for a particular test)

Reply:
Removed. Thanks :)